app/server.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const cookieParser = require('cookie-parser');
+const bodyParser = require('body-parser');
+const path = require("path");
+const healthzRouter = require("./routes/healthz.router");
+const productRouter = require("./routes/product.router");
+const scriptRouter = require("./routes/script.router");
+const appRouter = require("./routes/app.router");
+const fdkExtension = require("./fdk");
+const app = express();
+const config = require("./config");
+app.use(cookieParser("ext.session"));
+app.use(bodyParser.json({
+    limit: '2mb'
+}));
+app.get('/env.js', (req, res) => {
```

Comment:
what is the use of /env.js here ?

Reply:
added by boilerplate code. if required then env variables can be added here and used from the frontend side.